flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerRunner.java

```diff
@@ -331,6 +311,25 @@ public static void runTaskManager(Configuration configuration, ResourceID resour
 		taskManagerRunner.start();
 	}
 
+	public static void runTaskManagerSecurely(String[] args, ResourceID resourceID) {
+		try {
+			final Configuration configuration = loadConfiguration(args);
+
+			FileSystem.initialize(configuration, PluginUtils.createPluginManagerFromRootFolder(configuration));
+
+			SecurityUtils.install(new SecurityConfiguration(configuration));
+
+			SecurityUtils.getInstalledContext().runSecured((Callable<Void>) () -> {
```

Comment:
nit: can remove `(Callable<Void>)` I guess

Reply:
Right. I will remove the `(Callable<Void>)`.